src/Confluent.Kafka/Producer.cs

```diff
@@ -37,7 +37,7 @@ public class Producer : IDisposable
         private bool manualPoll;
         private bool disableDeliveryReports;
 
-        const int RD_KAFKA_PARTITION_UA = -1;
+        internal const int RD_KAFKA_PARTITION_UA = -1;
```

Comment:
internal because `Producer<TKey, TValue>` needs to know this. could possibly be better defined somewhere else (though I couldn't think where). bit ugly, but implementation detail which isn't publicly exposed.